data/endf8.py

```diff
@@ -39,16 +39,26 @@
 h5.attrs["last-resonance-group"] = fdy.get_default_group_structure().last_res_grp
 h5.attrs["library"] = lib_name
 
+# Define a helper function to process nuclides
+def process_nuclide(name: str, endf_filename: str, dilutions: list[float] | None = None):
```

Comment:
Do you think a similar function could be made for the TSls, where we just need to provide the TSL ENDF file and the list of temperatures ? We could also use ENDFtk to "extract" the tabulated temperatures from the ENDF file to simplify library generation... that might be a better idea to simplify the generation of libraries.